wallet/local/evm.go

```diff
@@ -0,0 +1,388 @@
+// Copyright (C) 2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package local
+
+import (
+	"fmt"
+	"math/big"
+	"net/url"
+	"strings"
+
+	"github.com/ava-labs/avalanchego/vms/platformvm/warp"
+
+	"github.com/ava-labs/avalanche-tooling-sdk-go/account"
+	"github.com/ava-labs/avalanche-tooling-sdk-go/evm"
+	"github.com/ava-labs/avalanche-tooling-sdk-go/evm/contract"
+	"github.com/ava-labs/avalanche-tooling-sdk-go/wallet"
+
+	ethereum "github.com/ava-labs/libevm"
+	ethCommon "github.com/ava-labs/libevm/common"
+	ethTypes "github.com/ava-labs/libevm/core/types"
+)
+
+// SetChain sets the current EVM chain by RPC URL
```

Comment:
This file looks good to me, however, I think since we don't provide options on evm client configurations (timeout, retries etc.), we should mention the defaults somewhere in doc comments so it's explicit

Reply:
added